README.rst

```diff
@@ -15,6 +15,15 @@ The `PyPA recommended`_ tool for installing Python packages.
 .. image:: https://readthedocs.org/projects/pip/badge/?version=latest
    :target: https://pip.pypa.io/en/latest
 
+.. image:: https://pypa.visualstudio.com/pip/_apis/build/status/Pip%20CI%20(Windows)
+   :target: https://pypa.visualstudio.com/pip/_build/latest?definitionId=6
```

Comment:
Is this badge showing the status of master or if the most recent build (eg a PR)?

Reply:
The badge will show the "default branch" for the build definition. Assuming you've configured your build definition's default branch to be the same one being checked by the PR build, you should be good to go. If you want to show a badge for a different branch, you can add `?branchName=<name>`.

Reply:
thanks @moswald, that's what I suspected.  master is the default branch here